### PR TITLE
migrate VerifyMicrosoftPackage.Facts.csproj to sdk csproj

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj [hunk 1]
```diff
@@ -1,82 +1,15 @@
...
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{FFF38843-B8E1-40E7-95E4-E3346599E8D5}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>NuGet.VerifyMicrosoftPackage.Facts</RootNamespace>
```
Comment (joelverhagen) [inline tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj:10]: The assembly name is changing, which is breaking out test script, I think.

File: tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj [hunk 2]
```diff
@@ -1,82 +1,15 @@
...
-      <PrivateAssets>all</PrivateAssets>
-    </PackageReference>
+    <ProjectReference Include="..\..\src\NuGetGallery.Services\NuGetGallery.Services.csproj" />
+    <ProjectReference Include="..\..\src\VerifyMicrosoftPackage\VerifyMicrosoftPackage.csproj" />
+    <ProjectReference Include="..\NuGetGallery.Core.Facts\NuGetGallery.Core.Facts.csproj" />
+    <ProjectReference Include="..\NuGetGallery.Facts\NuGetGallery.Facts.csproj" />
+
+    <PackageReference Include="xunit" Version="2.5.0" />
+    <PackageReference Include="xunit.analyzers" Version="1.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.5.0" PrivateAssets="all" />
```
Comment (erdembayar) [inline tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj:13]: <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>

**Review by erdembayar — APPROVED** [review]

File: tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj (hunk 1 above)
Comment (erdembayar) [reply tests/VerifyMicrosoftPackage.Facts/VerifyMicrosoftPackage.Facts.csproj:10]: It's breaking our CI build.
file not found: xxx\tests\VerifyMicrosoftPackage.Facts\bin\Release\VerifyMicrosoftPackage.Facts.dll

**Review by erdembayar — CHANGES_REQUESTED** [review]
Please check my comment for break CI build

**Pull request closed without merging**